func/code.func

```diff
@@ -16,6 +16,9 @@
 int workchains_equal?(slice addr1, slice addr2) asm "REWRITESTDADDR DROP SWAP REWRITESTDADDR DROP EQUAL";
```

Comment:
Special thanks could be removed if the hashing algorithm is replaced with ECVRF. Also, you can add yourself as a third author.